src/storage-client/src/controller.rs

```diff
@@ -175,9 +178,11 @@ impl<T> CollectionDescription<T> {
                         // No storage dependencies.
                     }
                 }
+                result.push(ingestion.remap_collection_id);
```

Comment:
We need to also express the dependency from the subsources (tables) of postgres sources to the remap shard, I think. Because those sub-sources have their own individual shard, with it's own individual upper/since.

Reply:
Makes sense! @petrosagg -- make sense to you? I checked those and they seem to have some kind of synchronization already, so am not totally clear what the mechanisms are in that case.